lib/fizzbuzz.rb

```diff
@@ -1,9 +1,13 @@
 class SuperFizzBuzz
 
   def run(input)
+    raise ArgumentError.new("Argument is not an Integer") unless input.instance_of? Fixnum
 
-    #Implement your code here
+    output = String.new
```

Comment:
It's preferred to use object literal notation for basic types. In other words, this:

``` ruby
output = ""
```